packages/utilities/psammead-locales/moment/ps.test.js

```diff
@@ -13,8 +13,8 @@ test('format', function () {
             ['MMMM Do YYYY, h:mm:ss',              'فبروري ۱۴ ۲۰۱۰، ۳:۲۵:۵۰'],
             ['M Mo MM MMMM MMM',                   '۲ ۲ ۰۲ فبروري فبروري'],
             ['L',                                  '۰۲/۱۴/۲۰۱۰'],
-            ['LL',                                 'فبروري ۱۴، ۲۰۱۰'],
-            ['LLL',                                'فبروري ۱۴، ۲۰۱۰ ۳:۲۵ PM'],
+            ['LL',                                 '۱۴ فبروري ۲۰۱۰'],
+            ['LLL',                                '۱۴ فبروري ۲۰۱۰ ۱۵:۲۵'],
             ['D MMMM YYYY',                        '۱۴ فبروري ۲۰۱۰'],
         ],
         b = moment(new Date(2010, 1, 14, 15, 25, 50, 125)),
```

Comment:
Wondering if it would be worth changing the variable names of just `a` and `b` here and in the following lines. It's understandable once I actually read into it but not at a glance. Was there any specific reason these were just called letters? If changes are to be made, it doesn't have to be in this PR. Maybe we can talk about it elsewhere first.

Reply:
On the lines above there is a comment: 
https://github.com/bbc/psammead/pull/3357/files#diff-f346ba0ea5f4f03a852d57caf66d4df3R6
```
// This asset overrides the gunit assertion done in the moment codebase.
// Format and styling of this file has been keep consistent with the official moment tests.
// An example of these tests can be seen at https://github.com/moment/moment/blob/develop/src/test/locale/en-gb.js
```
We haven't refactored the test structure, so we can contribute these suggestions back to the Moment codebase, after we believe they're complete.